replication-api-impl/src/main/java/com/connexta/replication/api/impl/data/ReplicationItemManagerImpl.java

```diff
@@ -149,7 +144,7 @@ public void remove(String id) {
         new GroupOptions().addGroupByField("metadata_id").setLimit(1).addSort(doneTimeDescSort);
```

Comment:
❓ Can there ever be more than one? Should it be ensured by the persistence layer that it never happens? Maybe by making it the unique id?

Reply:
Yes, but in this case we only care about the most recent since entities are sorted DESC by done time.

Reply:
My point is that there should never be more than one.

Reply:
Yes there should be because an metadata can have multiple items. For instance if there is a create, update, delete on the same metadata. All 3 entries will have the same metadata_id field.

Reply:
Sorry, I though I was looking at the filter index :-)